package.json

```diff
@@ -122,5 +122,8 @@
       "prettier --write",
       "git add"
     ]
+  },
+  "overrides": {
+    "micromatch": "4.0.8"
```

Comment:
What is this for?

Reply:
It's an internal dependency for quite a few libs in this repo : batfish, globby, jest, lint-staged, stylelint. Forcing this to this version where the security vulnerability has been fixed doesn't seem to affect anything for us and things work the same as before.